parsers/jscript.c

```diff
@@ -2097,7 +2112,9 @@ static bool parseStatement (tokenInfo *const token, bool is_inside_class)
 			/* Assume it's an assignment to a global name (e.g. a class) using
 			 * its fully qualified name, so strip the scope.
 			 * FIXME: resolve the scope so we can make more than an assumption. */
+			token->scopeKind = JSTAG_NONE;
 			vStringClear (token->scope);
```

Comment:
These two lines imply it is better to introduce a helper function named `clearTokenScope`.
```c
static void clearTokenScope(tokenInfo *token)
{
   vStringClear (token->scope);
   token->scopeKinde = KIND_GHOST_INDEX;
}
```

Reply:
`clearScope` or `resetScope` may be better.